tslib/src/functions/util-test-support-api/index.ts

```diff
@@ -0,0 +1,85 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions'
+import { performance } from 'perf_hooks'
+import config from '../../config'
+import { SchoolApi } from './school-api'
+import { UserApi } from './user-api'
+
+const functionName = 'util-test-support-api-create-school'
+
+function finish (start: number, context: Context): void {
+  const end = performance.now()
+  const durationInMilliseconds = end - start
+  const timeStamp = new Date().toISOString()
+  context.log(`${functionName}: ${timeStamp} run complete: ${durationInMilliseconds} ms`)
+}
+
+const httpTriggerFunc: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  if (!config.DevTestUtils.TestSupportApi) {
+    context.log('exiting as not enabled (default behaviour)')
+    return
+  }
+  // Respond in 230 seconds or the load balancer will time-out
+  // https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=javascript#limits
+  const start = performance.now()
+  const entity = context.bindingData.entity
```

Comment:
maybe i've not had enough coffee yet, but how is this entity property populated?  the only binding we have is an incoming http request 🤔

Reply:
The `entity` bit is defined as an alpha in the `route` property of the function.json. 

`      "route": "test-support/{entity:alpha}"`

Reply:
ah yes, overlooked the route stuff in the function definition. thanks!